### ISSUE TITLE
Installation instructions on your websites are outdated

### ISSUE THREAD

**Issue by Valdegg** [urllib3/urllib3#2578]
It says:

"Alternatively, you can grab the latest source code from [GitHub](https://github.com/urllib3/urllib3):

$ git clone git://github.com/urllib3/urllib3.git
$ python setup.py install"

While the python command doesn't work any more. The correct version is on the readme on your github page though. Just wanted to let you know - I needed to reinstall urllib3 and this was a problem.

**Comment by pquentin** [comment]
Thanks for the report!

We fixed that already: https://github.com/urllib3/urllib3/commit/d662fcd9301b94a413a0d21c6c287268cd082da1 but it was not released yet.

However, when following those instructions you're installing the 2.0 development which is not ready for public use yet. I guess we should add a note in the 1.26.x docs to checkout the 1.26.x branch before installing.

**Comment by Pradhvan** [comment]
@pquentin Hey 👋🏾  in the docs section of the branch`1.26x` 

How about adding a step to switch to `1.26x` branch instead of a note? 

In the [installation section](https://github.com/urllib3/urllib3/blob/1.26.x/docs/index.rst#installing)

```
$ git clone https://github.com/urllib3/urllib3.git
$ cd urllib3
$ git checkout 1.26.x  
$ pip install .
```

**Comment by pquentin** [comment]
Yes, that would be a good idea!

**Comment by Pradhvan** [comment]
@pquentin cool 👍🏾 will add a PR for this. Do you want the exact instructions in any other branch or just branch `1.26.x` is fine?

**Comment by pquentin** [comment]
It's fine, 1.26.x and main are the only branches we care about.

Thank you!

**Comment by Pradhvan** [comment]
@pquentin sorry for the delay and the some git mess above. But finally created the PR. 😄

**Comment by pquentin** [comment]
No worries, thanks for stepping up to fix this!

**Comment by Pradhvan** [comment]
@pquentin no problem. Thank you for helping out updating a change for readme. I had forgotten about it. 😄

**Comment by sethmlarson** [comment]
Closed in #2643
